pkg/commands/product/bot_management/status.go

```diff
@@ -0,0 +1,112 @@
+package bot_management
```

Comment:
The linter is agreeing with me that package names shouldn't have underscores.

Reply:
Yeah... I've been trying to stick with using our actual product IDs for ease of understanding by future maintainers, but the Go community has a different opinion (except for test packages, where underscores are allowed).

Reply:
In that case it is representing a build tag. The same works for other specific words that match OS targets or architectures, like `_linux` or `_arm64`.

Reply:
There also the only exception to two packages in one directory, where `p` and `p_test` can coexist to allow for blackbox testing of `p`.